### PR TITLE
Use pre-converted images on ghcr.io

### PR DESCRIPTION
**Pull request by ktock**
Fixes: #155

This commit uses pre-converted images stored on ghcr.io in our Github Actions
jobs. This also updates some docs in this repo to use images on ghcr.io.

### CODE REVIEW

File: docs/pre-converted-images.md [hunk 1]
```diff
@@ -18,19 +24,19 @@ In the following table, `Image Name` column contains placeholders `{{ suffix }}`
...
-|`docker.io/stargz/jruby:9.2.8.0-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/node:13.13.0-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/perl:5.30-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/php:7.3.8-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/pypy:3.5-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/python:3.7-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/r-base:3.6.1-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/redis:5.0.5-{{ suffix }}`|Code execution until up and ready message (`Ready to accept connections`) is printed|
-|`docker.io/stargz/rethinkdb:2.3.6-{{ suffix }}`|Code execution until up and ready message (`Server ready`) is printed|
+|`{{ repository }}/alpine:3.10.2-{{ suffix }}`|Executing `echo hello` on the shell|
```
Comment (AkihiroSuda) [inline docs/pre-converted-images.md:27]: This is hard to read. Let's list actual image refs.

Comment (ktock) [reply docs/pre-converted-images.md]: Thanks for the comment. Listed the actual refs.

File: docs/pre-converted-images.md [hunk 2]
```diff
@@ -18,19 +20,51 @@ In the following table, `Image Name` column contains placeholders `{{ suffix }}`
...
-|`docker.io/stargz/jruby:9.2.8.0-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/node:13.13.0-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/perl:5.30-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/php:7.3.8-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/pypy:3.5-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/python:3.7-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/r-base:3.6.1-{{ suffix }}`|Printing `hello`|
-|`docker.io/stargz/redis:5.0.5-{{ suffix }}`|Code execution until up and ready message (`Ready to accept connections`) is printed|
-|`docker.io/stargz/rethinkdb:2.3.6-{{ suffix }}`|Code execution until up and ready message (`Server ready`) is printed|
+|`ghcr.io/stargz-containers/alpine:3.10.2-org`|*none*|
```
Comment (AkihiroSuda) [inline docs/pre-converted-images.md:23]: Not none

Comment (ktock) [reply docs/pre-converted-images.md:23]: Should it be just empty or "NA"?

Comment (AkihiroSuda) [reply docs/pre-converted-images.md:23]: Empty or "Same as above"

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**